### ISSUE TITLE
get-tppcertificate JKS

### ISSUE THREAD

**Issue by Curtmcgirt** [gdbarron/VenafiTppPS#147]
# Environment

```
Operating System:
VenafiTppPS version:
PowerShell version:
```

# Steps to reproduce

when i use "-format JKS" wiith get-tppcertificate, i don't see an option for 'keystorepassword", and i get an error saying keystorepassword is missing even if i use the -securepassword switch. 

# Expected behavior

get a jks file

"400 Missing KeystorePassword.: {"Error":"Missing KeystorePassword."}
At C:\Program Files\WindowsPowerShell\Modules\VenafiTppPS\2.2.0\Public\Invoke-TppRestMethod.ps1:154 char:17
+ ...             throw ('"{0} {1}: {2}' -f $originalError.Exception.Respon ...
+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : OperationStopped: ("400 Missing Ke...rePassword."}
:String) [], RuntimeException
    + FullyQualifiedErrorId : "400 Missing KeystorePassword.: {"Error":"Missing KeystorePassword."}

**Comment by Curtmcgirt** [comment]
seems like the 'expected behavior' and 'what's actually happening' fields are joined together. also i had to put my description into "steps to reproduce" because it did not appear anywhere on this page when i put it in the 'description' at the top.

**Comment by gdbarron** [comment]
I had never added the Keystore password, but have now.  `-SecurePassword` was used for the private key.  I've added `-KeystorePassword` and renamed `-SecurePassword` to `-PrivateKeyPassword`, including an alias.  I also removed the need for `-IncludePrivateKey` and the password is the only param needed.  I also updated the param sets to enforce `-FriendlyName` being provided when using `-KeystorePassword`.  All the help updated and added KeystorePassword to `Write-VerboseWithSecret` so the password isn't shown when using verbose.  It's in v2.2.4.  Let me know of any issues.
